src/components/collapsible/styles/collapsibleInfoBox.scss

```diff
@@ -0,0 +1,10 @@
+$info-icon-width: 20px;
+$collapsible-border-radius: 4px;
```

Comment:
There is already a border radius variable in vapor and it is not set to 4. You should import vapor's variables instead.

Reply:
Took that value out of UX's mockup. How do we import vapor variables in css modules?

Reply:
You can look at the DiffViewer, I've imported vapor's variables